modules/migration-migrating-on-prem-to-cloud.adoc

```diff
@@ -0,0 +1,114 @@
+* It must be possible to create a load balancer on the destination cluster. Refer to your cloud provider to ensure this is possible.
+* Have names prepared to assign to namespaces, on both the source cluster and the destination cluster, in which to run the VPN tunnel. These namespaces should not be created in advance. For information on namespace rules, see \https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names.
+* When connecting multiple firewall-protected source clusters to the cloud cluster, each source cluster requires its own namespace.
+* OpenVPN server is installed on the destination cluster.
+* OpenVPN client is installed on the source cluster.
+* When configuring the source cluster in {mtc-short}, the API URL takes the form of \https://proxied-cluster.${namespace}.svc.cluster.local:8443.
+** If you use the API, see _Create a MigCluster CR manifest for each remote cluster_.
+** If you use the {mtc-short} web console, see _Migrating your applications using the {mtc-short} web console_.
+* The {mtc-short} web console and Migration Controller must be installed on the target cluster.
+
+.Procedure
+
+. Install the `crane` utility:
++
+[source,terminal,subs="+quotes"]
+----
+$ podman cp $(podman create registry.redhat.io/rhmtc/openshift-migration-controller-rhel8:v1.7.0):/crane ./
+----
+
+. Log in remotely to a node on the source cluster and a node on the destination cluster.
+
+. Obtain the cluster context for both clusters after logging in:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc config view
+----
+
+. Establish a tunnel by entering the following command on the command system:
++
+[source,terminal,sub="+quotes"]
+----
+$ crane tunnel-api [--namespace _<namespace>_] \
+      --destination-context _<destination-cluster>_ \
+      --source-context _<source-cluster>_
+----
++
+If you don't specify a namespace, the command uses the default value `openvpn`.
++
+For example:
++
+[source,terminal,subs="+quotes"]
+----
+$ crane tunnel-api --namespace my_tunnel \
+      --destination-context openshift-migration/c131-e-us-east-containers-cloud-ibm-com/admin \
+      --source-context default/192-168-122-171-nip-io:8443/admin
+----
++
+The command generates SSL Certificates. This process might take several minutes. A message appears when the process completes.
++
+The OpenVPN server starts on the destination cluster and the OpenVPN client starts on the source cluster.
++
+After a few minutes, the load balancer resolves on the source node.
++
+[TIP]
+====
+You can view the log for the OpenVPN pods to check the status of this process by entering the following commands with root privileges:
+
+[source,terminal,subs="+quotes"]
+----
+# oc get po -n _<namespace>_
+NAME            READY     STATUS      RESTARTS    AGE
+_<pod_name>_    2/2       Running     0           44s
+# oc logs -f -n _<namespace>_ _<pod_name>_ -c openvpn
+----
```

Comment:
```suggestion
[source,terminal,subs="+quotes"]
----
# oc get po -n <namespace>
----

.Example output
[source,terminal]
----
NAME            READY     STATUS      RESTARTS    AGE
<pod_name>    2/2       Running     0           44s
----

[source,terminal]
----
# oc logs -f -n <namespace> <pod_name> -c openvpn
----
```

The two commands and the example output needs to be in separate source blocks. Re: [Code blocks, command syntax, and example output](https://github.com/openshift/openshift-docs/blob/9a12c390cfe11f474e7234b0601b74fa320e6654/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) in the repo guidelines.

See the comment above re: italics for variables.

Reply:
At the risk of giving a technical review, is it `oc get po` vs. `oc get pod`?

Reply:
You might need `+` signs in between the commands if it does not render correctly.